magma/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: "3.4"
+volumes:
+  magma-app-gems:
+    driver: local
+  magma-db-data:
+    driver: local
+  magma-log-dir:
+    driver: local
+  magma-tmp-dir:
+    driver: local
+
+x-magma_base: &magma_base
+  build:
+    context: .
+    dockerfile: docker/app/Dockerfile.development
+  image: magma_base
+  volumes:
+    - .:/app:cached
+    - magma-app-gems:/app/vendor/bundle:rw
+    - magma-log-dir:/app/log:rw
+    - magma-tmp-dir:/app/tmp:rw
+    - ./projects:/app/projects
+    - ../etna:/etna:cached
+  environment: &app_environment
+    COVERAGE: "true"
+    DATABASE_HOST: "db"
+    magma_ENV: "development"
```

Comment:
Does this need to be `MAGMA_ENV` instead of `magma_ENV`?

Reply:
Search and replace artifact, yup.